app/views/articles/_user_metadata.html.erb

```diff
@@ -43,43 +43,47 @@
           </div>
         </li>
       <% end %>
-      <% if @user.employment_title.present? %>
-        <li>
-          <div class="key">
-            Work
-          </div>
-          <div class="value">
-            <%= @user.employment_title %>
-            <% if @user.employer_name.present? %>
-              <span class="opacity-50"> at </span>
-              <% if @user.employer_url.present? %>
-                <a href="<%= @user.employer_url %>" target="_blank" rel="noopener"><%= @user.employer_name %></a>
-              <% else %>
-                <%= @user.employer_name %>
+      <% if ForemInstance.dev_to? %>
+        <% if @user.employment_title.present? %>
+          <li>
+            <div class="key">
+              Work
+            </div>
+            <div class="value">
+              <%= @user.employment_title %>
+              <% if @user.employer_name.present? %>
+                <span class="opacity-50"> at </span>
+                <% if @user.employer_url.present? %>
+                  <a href="<%= @user.employer_url %>" target="_blank" rel="noopener"><%= @user.employer_name %></a>
+                <% else %>
+                  <%= @user.employer_name %>
+                <% end %>
               <% end %>
-            <% end %>
-          </div>
-        </li>
+            </div>
+          </li>
+        <% end %>
       <% end %>
-      <% if @user.location.present? %>
+      <% if @user.profile.location.present? %>
         <li>
           <div class="key">
             Location
           </div>
           <div class="value">
-            <%= @user.location %>
+            <%= @user.profile.location %>
           </div>
         </li>
       <% end %>
-      <% if @user.education.present? %>
-        <li>
-          <div class="key">
-            Education
-          </div>
-          <ul class="value">
-            <%= @user.education %>
-          </ul>
-        </li>
+      <% if ForemInstance.dev_to? %>
```

Comment:
@benhalpern Same question as above.

Reply:
Yeah, definitely don't need to rely on anything special about DEV here in this case.

Reply:
@benhalpern Again, trying to ensure we fully are in agreement about the ramifications: if we don't want to special case this for DEV we would be completely removing it as there's no guarantee that Forem instances have a profile field like this or that it has this exact name.

Reply:
@citizen428 — Yes, `education` can be entirely optional and functionally equivalent to `astrology_sign` for a Zodiac Forem or `favorite_team` for an NBA Forem.

***

(As a total aside, we may want to make room for a "select input" type of profile field because my above examples would both call for a finite number of options— There could definitely be a lot of value in profile fields which act as structured data instead of free text.)

Reply:
@benhalpern Yes, we could add that. I may actually open up a stage 0 RFC for this though as we may want to think about how to store the options.